test/acsengine.deploy

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env groovy
+
+node {
+  withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'AZURE_CLI_SPN_ACS_TEST',
```

Comment:
Does the test runner provide credentials at the beginning of a test run? If not how are the credentials retrieved?